ydb/docs/en/core/reference/ydb-cli/export-import/_includes/tools_dump.md

```diff
@@ -27,6 +27,8 @@ The `tools dump` command dumps the database data and objects schema to the clien
 
 `--save-partial-result`: Don't delete the result of partial dumping. Without this option, the dumps that terminated with an error are deleted.
 
+`--preserve-pool-kinds`: Preserve storage pool kind settings. If this option is enabled, storage pool kind will be saved to dump. In this case, if there will be no such storage pool kind in database on restore, error will occur. By default this option is disabled and any existing storage pool kind will be used on restore.
```

Comment:
```suggestion
`--preserve-pool-kinds`: If this option is enabled, the `tools dump` command saves storage pool names to the dump. To import such a dump, the names of storage pools in the database must match the names saved in the dump. If at least one storage pool with the required name is missing, the import procedure will end with an error. By default this option is disabled, and the import procedure will use any existing storage pools regardless of their names.
```

Reply:
I suggest it's necessary to use 'storage pool kind' instead of 'storage pool name', because it's the way they're referenced in the configuration: https://ydb.tech/docs/en/reference/configuration/#syntax3 

changed the rest of the description as suggested though